projects/ng-dynamic-component/src/lib/dynamic.component.ts

```diff
@@ -41,6 +42,8 @@ export class DynamicComponent<C = unknown>
   ndcDynamicProviders?: StaticProvider[] | null;
   @Input()
   ndcDynamicContent?: Node[][];
+  @Input()
+  ndcDynamicModule?: NgModuleRef;
```

Comment:
```suggestion
  ndcDynamicNgModule?: NgModuleRef<unknown>;
```

Reply:
Also you forgot to import `NgModuleRef` and [the build has failed](https://ci.appveyor.com/project/gund/ng-dynamic-component/builds/46014992#L43).

Reply:
fixed